db/db.go

```diff
@@ -42,6 +42,7 @@ type template interface {
 	DeleteTemplate(ctx context.Context, name string) error
 	ListTemplates(in string, fn func(id, n string, in, del *timestamp.Timestamp) error) error
 	UpdateTemplate(ctx context.Context, name string, data string, id uuid.UUID) error
+	ListTemplateRevisions(id string, fn func(id string, revision int, tCr *timestamp.Timestamp) error) error
```

Comment:
Let's try to write something a bit more readable:

```go
ListRevisionsByTemplateID(templateID string, fn func(revisionID string, revision int, tCr *timestamp.Timestamp) error) error
```

I am not sure what `revisiononID string` and `revision int` are